### DISCUSSION TITLE
Power and voltage limit reached. Disabling automatic power control

### DISCUSSION THREAD

**Discussion by Max8409** [hoylabs/OpenDTU-OnBattery#1078] [Q&A]
In Verbindung mit dem Huawei AC Netzteil. Weiß jemand, wodurch diese Meldung verursacht wird und vorallendingen wie man es wieder aktiviert nachdem der Fehler aufgetreten ist?
Vielen Dank!

**Comment by tstrempel1980** [comment]
Ich habe gerade das selbe Problem, keine Ahnung woran das liegt.
Das Gerät hängt an einer shelly steckdose, die reboote ich, dann und es geht wieder.

woher kommt das Limit?
[Huawei::Controller] Power and voltage limit reached. Disabling automatic power control.
